Cargo.toml

```diff
@@ -21,6 +21,7 @@ winapi = { version = "0.2.5", optional = true }
 dbghelp-sys = { version = "0.2", optional = true }
 cfg-if = "0.1"
 rustc-demangle = "0.1"
+shared_library = { version = "0.1.5", optional = true }
```

Comment:
It looks like this is just used for one macro, perhaps dlopen/dlsym could be called manually?